src/backends/tensorrt/tensorrtmodel.h

```diff
@@ -67,7 +67,7 @@ namespace dd
     std::string _def;
     std::string _weights;
     bool _has_mean_file = false;
-    std::string _source_type = "caffe"; // or "onnx"
+    std::string _source_type; //"caffe" or "onnx"
```

Comment:
Can we still provide the tensorRT engine directly? Some libraries can export tensor engines directly (e.g. torch2trt?) and I know it was possible to import the engine in dede.

Reply:
Yes for Caffe, e.g. `net_tensorRT.proto` and any `.onnx` file should work out of the box now.